frontend/src/concepts/pipelines/content/tables/pipeline/PipelinesTableExpandedRow.tsx

```diff
@@ -13,6 +13,7 @@ import { PipelineKF, PipelineRunKF } from '~/concepts/pipelines/kfTypes';
 import IndentSection from '~/pages/projects/components/IndentSection';
 import { FetchState } from '~/utilities/useFetchState';
 import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { TABLE_CONTENT_LIMIT } from '~/const';
```

Comment:
This is Pipeline specific, no need to make this global. We should keep things isolated as much as we can. 

Perhaps use `~/concepts/pipelines` instead?